releases/release-1.31/release-notes/maps/pr-125488-map.yaml

```
@@ -0,0 +1,4 @@
+pr: 125488
+releasenote:
+  text: 'Dynamic Resource Allocation (DRA): Added new API and several new features.'
```

(do we know what the new API kind / new other thing is named?)

---

@pohly Can you please provide elaborated version of the user-facing notes? I did take a look at the enhancement but I think it makes more sense for you to decide what needs to be presented to the users.

---

slack thread for reference: https://kubernetes.slack.com/archives/CN1KH4K9A/p1722972348572189?thread_ts=1722709505.215119&cid=CN1KH4K9A

how does this sound:
```suggestion
  text: 'Enhanced the Dynamic Resource Allocation (DRA) with a new version of the resource.k8s.io API group. While the ResourceClaim remains the primary user-facing type, many changes have been made, leading to a new version, v1alpha3, which is not compatible with the previous version.'
```

---

thanks @npolshakova for a better version:
```suggestion
  text: 'Enhanced the Dynamic Resource Allocation (DRA) with an updated version of the resource.k8s.io API group. The primary user-facing type remains the ResourceClaim, however significant changes have been made, resulting in the new version, v1alpha3, which is not compatible with the previous version.'
```